src/git/mod.rs

```diff
@@ -175,7 +175,7 @@ pub fn remove_history(project_dir: &Path) -> io::Result<()> {
 
                 if e.to_string().contains("The process cannot access the file because it is being used by another process.") {
                     let wait_for = Duration::from_secs(2_u64.pow(attempt.sub(1).into()));
-                    warn!("Git history cleanup failed with a windows process blocking error. [Retry in {:?}]", wait_for);
+                    warn!("{} Git history cleanup failed with a windows process blocking error. [Retry in {:?}]", emoji::WARN, wait_for);
```

Comment:
Instead of prefixing the emoji like this, I recommend to configure the env_logger in a way that each log level would include those prefixes.

You can see an [example in the rust cookbook](https://rust-lang-nursery.github.io/rust-cookbook/development_tools/debugging/config_log.html#include-timestamp-in-log-messages)

Reply:
Good point! I know there is a WARN emoji but how would I map other log levels to the defined emoji's here?:
https://github.com/cargo-generate/cargo-generate/blob/main/src/emoji.rs#L5
because there are times when info is used with non-info like sparkle:
https://github.com/cargo-generate/cargo-generate/blob/main/src/favorites.rs#L55

Reply:
Maybe it's a good opportunity to align them. Or if that's too noisy maybe only warning and error are good candidates for now.

Reply:
Great, I will take that just integrates warn|error emoji levels at the formatter.

Reply:
First shot at addressing it here: cf8f6e6e70b596876a67e4bc58817b16781f7cfc

Reply:
@sassman is there anything else I should address?

Reply:
thanks for addressing this! Looks very good so far.